src/Drupal/ContentTrait.php

```
@@ -57,6 +60,47 @@ public function contentDelete(string $type, TableNode $table): void {
+   * @code
+   * Given the following page content with fields:
+   *   | title  | [TEST] Page 1        | [TEST] Page 2        |
+   *   | body   | First page content   | Second page content  |
+   *   | status | 1                    | 1                    |
+   * @endcode
+   */
+  public function contentCreateWithFields(string $type, TableNode $table): void {
+    $entities = $this->helperTransposeVerticalTable($table);
+
+    // Convert to the format expected by createNodes().
+    $horizontal_table = new TableNode([]);
```

PHPMD (2.15.0)

The variable $horizontal_table is not named in camelCase. (undefined)

(CamelCaseVariableName)